Dockerfile.ci

```diff
@@ -1375,8 +1375,8 @@ RUN echo "Airflow version: ${AIRFLOW_VERSION}"
 # Those are additional constraints that are needed for some extras but we do not want to
 # force them on the main Airflow package. Currently we need no extra limits as PIP 23.1+ has much better
 # dependency resolution and we do not need to limit the versions of the dependencies
-# !!! MAKE SURE YOU SYNCHRONIZE THE LIST BETWEEN: Dockerfile, Dockerfile.ci
```

Comment:
Does removing this comment mean that we don't need to synchronize with Dockerfile anymore?

Reply:
Yes. I removed it long time ago -it's not been used there, only now I realized the comment is still here.

Reply:
Or not.... I SHOULD HAVE REMOVED IT.

Reply:
I will remove it in the subsequent PR from `Dockerfile`